scripts/end_to_end_test/end_to_end_test.yaml

```diff
@@ -190,3 +190,19 @@ steps:
   - sleep: 5
   - run_python_test: { "script": ".smart_contract_tests.init_owner_address" }
 ...
+# End-to-end test DMLF scenarios
```

Comment:
I put the test here: only this yaml file will be read by the CI, sorry about the confusion.